### PR TITLE
Add contact privacy notice page

### LINKED ISSUE #3123: Privacy notices - Contact Privacy Notice - Build
Copydoc: https://docs.google.com/document/d/1kQmhTCRSnx05oYEFcNr0Dq7UUFrznq-GvQlRxvNAZ9s/edit?usp=sharing_eil&ts=5af1c29d

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add contact privacy notice page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/legal/dataprivacy/contact>
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1kQmhTCRSnx05oYEFcNr0Dq7UUFrznq-GvQlRxvNAZ9s/edit?ts=5af1c29d)

## Issue / Card
Fixes #3123

### CODE REVIEW

File: templates/legal/dataprivacy/contact.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <p>Version - April 2018</p>
+      <p>This privacy notice tells you about the information we collect from you when you sign up to receive our regular newsletter via our website. In collecting this information, we are acting as a data controller and, by law, we are required to provide you with information about us, about why and how we use your data, and about the rights you have over your data.</p>
+      <h2 id="who-are-we">Who are we?</h2>
+      <p>We are Canonical Group Limited. Our address is 5th Floor Blue Fin Building, 110 Southwark Street, London, SE1 0SU. You can contact us by post at the above address or by email to <a href="mailto:dataprotection@canonical.com">dataprotection@canonical.com</a> for the attention of &ldquo;Legal&rdquo;.</p>
+      <p>We are not required to have a data protection officer, so any enquiries about our use of your personal data should be addressed to the contact details above.</p>
+      <h2 id="what-personal-data-do-we-collect">What personal data do we collect?</h2>
+      <p>When you subscribe to our newsletter, we ask you for your name and your email address and related information.</p>
```
Comment (Caleb-Ellis) [inline templates/legal/dataprivacy/contact.html]: This does not match the copydoc

File: templates/legal/dataprivacy/contact.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <h2 id="why-do-we-collect-this-information">Why do we collect this information?</h2>
+      <p>We will use your information to send you our newsletter, which contains information about our products. This may involve us calling you, where we have your phone number in order to do so.</p>
```
Comment (Caleb-Ellis) [inline templates/legal/dataprivacy/contact.html]: Same here

File: templates/legal/dataprivacy/contact.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <p>We ask for your consent to do this, and we will only send you our newsletter for as long as you continue to consent.</p>
+      <p>Where you have agreed, we will also use your information to send you other news and product updates from Canonical. This may involve us calling you, where we have your phone number in order to do so.</p>
```
Comment (Caleb-Ellis) [inline templates/legal/dataprivacy/contact.html]: Same here

**Comment by steverydz** [comment]
@Caleb-Ellis Updated

**Comment by frankiegranato** [comment]
One small text edit - sorry @steverydz

**Comment by steverydz** [comment]
@frankiegranato Done

**Comment by frankiegranato** [comment]
Looks like the ampersand is still there :-(

**Comment by steverydz** [comment]
@frankiegranato Ah, I forgot to rebuild the demo - try again :)

**Pull request merged**